<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="acc_man_wid" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>240</width>
      <height>160</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="book_man_wid" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>240</width>
      <height>160</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="rtn_man_wid" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>240</width>
      <height>160</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="ban_man_wid" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>180</y>
      <width>240</width>
      <height>160</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
